test/dlc_tests/container_tests/bin/pytorch_tests/testTorchdata

```diff
@@ -12,7 +12,7 @@ echo "Fetch Torchdata S3 IO test script."
 TORCHDATA_RELEASE_TAG=$(python -c "import torchdata as td; print(td.__version__.split('+')[0])")
 git clone $TORCHDATA_REPO \
     && pushd data \
-    && git checkout tags/v${TORCHDATA_RELEASE_TAG} -b ${TORCHDATA_RELEASE_TAG} \
+    && git checkout ce075397428c90be312f87b60d4701b2e3ff3700 \
```

Comment:
let's add a comment on why we are fixing the commit

Reply:
done

Reply:
Question. Why are we locking to a specific hash, instead of matching the branch to that of the PT version we are dealing with? Has this been tested against all supported versions of PyTorch?

Reply:
@xncqr The release tag for v0.7.0 is missing (only the RC tag exists), so the original code doesn't work with PT2.1.0. We decided to just use a fixed commit on main that is working, and yes I tested it against the beta/RC image for PT2.1 ec2 inference (training isn't available yet), as well as 2.0.1. I believe 1.13.1 will be deprecated with the release of 2.1.0?

Reply:
PT 1.13 is still supported and and would be good to verify this test will works with that version.